packages/apps/storybook/.storybook/preview.js

```diff
@@ -4,6 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 import addons from "@storybook/addons";
 import { themes } from "@storybook/theming";
+import "@itwin/itwinui-react/styles.css";
```

Comment:
With the new version of iTwinUI, we should probably use global decorators so the inheritance can work correctly (https://storybook.js.org/docs/6.2/react/writing-stories/decorators#global-decorators)

So, instead of `document.body.dataset.iuiTheme` (which was 1.0 "how to", we should have a 
```
export const decorators = [
  (Story) => {
    const [dark, setDark] = React.useState(false)
    React.useEffect(() => channel.on("DARK_MODE", setDark));
    <ThemeProvider theme={dark ? "dark" : "light"} ><Story /></ThemeProvider>
  },
];
```

Reply:
Done